SignalServiceKit/src/Util/NSDate+OWS.m

```diff
@@ -3,17 +3,15 @@
 //
 
 #import "NSDate+OWS.h"
-#import <chrono>
 
 NS_ASSUME_NONNULL_BEGIN
 
 @implementation NSDate (millisecondTimeStamp)
 
 + (uint64_t)ows_millisecondTimeStamp
 {
-    uint64_t milliseconds
-        = (uint64_t)(std::chrono::system_clock::now().time_since_epoch() / std::chrono::milliseconds(1));
-    return milliseconds;
+    NSDate *now = [self new];
+    return (uint64_t)(now.timeIntervalSince1970 * 1000);
```

Comment:
Huh - we should use `chrono` where we can; it's much higher precision.  Why can't `chrono` be used in the SAE?

Reply:
Please add a `FIXME SHARINGEXTENSION` here if you're not going to resolve this in this PR.